app/src/components/settings/settings_view/index.tsx

```diff
@@ -51,14 +30,23 @@ const TextLighter = styled.p`
   line-height: 14.06px;
   margin: 0;
 `
-
-const ButtonRow = styled.div`
+const ButtonContainer = styled.div`
   display: flex;
-  margin-left: auto;
-
-  button:first-child {
-    margin-right: 12px;
-  }
+  flex-direction: row;
+  justify-content: center;
+  width: 364px;
+  padding: 0px;
+  margin-top: 24px;
+  height: 100%;
+`
+const SetAndSaveButton = styled(ButtonRound)`
```

Comment:
Can you see how I did deposit and withdraw buttons and base it on that....You are hardcoding widths again here...

Reply:
🤯 
it worked
TIL `flex:1` 
lol